lib/livebook_web/live/app_helpers.ex

```diff
@@ -301,6 +301,6 @@ defmodule LivebookWeb.AppHelpers do
     Enum.reject(apps, &(&1.slug == app.slug))
   end
 
-  defp zta_help(%{help: help}), do: "#{help}"
-  defp zta_help(_), do: nil
+  defp zta_placeholder(%{placeholder: placeholder}), do: "#{placeholder}"
```

Comment:
Isn't it always a string?

```suggestion
  defp zta_placeholder(%{placeholder: placeholder}), do: placeholder
```